mission_planner3/UAVState[] other_UAVs_state